app/assets/javascripts/discourse/app/app.js

```diff
@@ -1,5 +1,5 @@
-import "./global-compat";
 import "./loader-shims";
+import "./global-compat";
```

Comment:
I think this didn't ended up being necessary, but it seems a bit more correct anyway